web/styles/app_variables.css

```diff
@@ -274,6 +274,9 @@
         (var(--legacy-body-line-height-unitless) * 1em) + 0.3571em
     );
     --right-sidebar-left-spacing: 5px;
+    /* Used in both left and right sidebar at the bottom, to allow
+       the user to scroll past the bottom-most elements a little bit. */
+    --sidebar-bottom-spacing: 1.4286em; /* 20px / 14px  em */
```

Comment:
I was copying the format above and below in this file. Is it better to change this line? I think it would be better if one of us returns later to redo them all .